tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/ApalacheOper.scala

```diff
@@ -112,4 +112,48 @@ object ApalacheOper {
 
     override val precedence: (Int, Int) = (100, 100)
   }
+
+  /**
```

Comment:
I wonder how it looks when you press `Alt+J` in Intellij. You should probably use javadoc formatting.

Reply:
Not sure what `Alt+J` does on your machine. I'm not too worried about comment formatting, because the in-code documentation is never really complied to JavaDoc, but is mostly looked at in source.

Reply:
@shonfeder what are your thoughts about that?

Reply:
No opinion on comment formatting. Being able to publish api docs at some point is probably a good idea tho. As long as the formatting is consistent due to the auto formatting and we have the option to generate and compile the app docs if we want, seems ok to me.

Reply:
right. My problem is that the comments will not be properly formatted. Most likely, the comment under discussion will produce a mess of text. That's why I suggested Jure to try it out. The only problem is that `Ctrl-j` is the right shortcut, not `Alt-j` :-)

Reply:
If there are no more comments on the code in this PR I'm going to close it. I'd suggest opening an issue for "scaladoc generation" as part of the pipeline, if you feel that's important.